src/System.Runtime/tests/System/Text/StringBuilderTests.netcoreapp.cs

```diff
@@ -199,5 +199,84 @@ public static void Insert_CharSpan_Invalid()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("index", () => builder.Insert(builder.Length + 1, new ReadOnlySpan<char>(new char[0]))); // Index > builder.Length
             AssertExtensions.Throws<ArgumentOutOfRangeException>("requiredLength", () => builder.Insert(builder.Length, new ReadOnlySpan<char>(new char[1]))); // New length > builder.MaxCapacity
         }
+
```

Comment:
I don't see Append of itself. Both overloads allow it, 2nd one if index and count are 0.

Reply:
I have added append of itself and 2nd one is already there
https://github.com/Anipik/corefx/blob/9cfbd0f74f843ea2934d3b85c69ebbb2fcd11fa6/src/System.Runtime/tests/System/Text/StringBuilderTests.netcoreapp.cs#L242